goodkey/blocked.go

```diff
@@ -52,18 +67,38 @@ func loadBlockedKeysList(filename string) (*blockedKeys, error) {
 	}
 
 	var list struct {
-		BlockedHashes []string `yaml:"blocked"`
+		BlockedHashes    []string `yaml:"blocked"`
+		BlockedHashesHex []string `yaml:"blockedHashesHex"`
 	}
 	if err := yaml.Unmarshal(yamlBytes, &list); err != nil {
 		return nil, err
 	}
 
-	if len(list.BlockedHashes) == 0 {
+	if len(list.BlockedHashes) == 0 && len(list.BlockedHashesHex) == 0 {
 		return nil, errors.New("no blocked hashes in YAML")
 	}
 
-	blockedKeys := make(blockedKeys, len(list.BlockedHashes))
-	for _, hash := range list.BlockedHashes {
+	blockedKeys := make(blockedKeys, len(list.BlockedHashes)+len(list.BlockedHashesHex))
+	for _, b64Hash := range list.BlockedHashes {
+		var hash keyHash
+		n, err := base64.StdEncoding.Decode(hash[:], []byte(b64Hash))
```

Comment:
This and below will panic if `DecodedLen(b64Hash) > 32`.

Reply:
Good catch, thanks!